Exiled.Events/EventArgs/Player/EscapingEventArgs.cs

```diff
@@ -53,6 +87,11 @@ public EscapingEventArgs(Player player, RoleTypeId newRole, EscapeScenario escap
         /// </summary>
         public EscapeScenario EscapeScenario { get; set; }
 
+        /// <summary>
+        ///     Gets the TicketsToChange that will represent the amount of tickets granted after the player escapes.
+        /// </summary>
+        public Dictionary<SpawnableTeamType, float> TicketsToChange { get; }
```

Comment:
That's get only.

Reply:
Oh ok, then just do `get; set;`? Because that indicates the property can be edited.